server/server.go

```diff
@@ -115,7 +115,8 @@ func NewServer(ctx *Context) (*Server, error) {
 
 	s.kvDB = kv.NewDBServer(sender)
 	s.kvREST = kv.NewRESTServer(s.kv)
-	s.node = NewNode(s.kv, s.gossip)
+	// TODO(bdarnell): make StoreConfig configurable.
```

Comment:
We'll definitely want to change this in production - single-datacenter deployments will want much smaller election timeouts than geographically-distributed ones.